app/script/components/list/conversationListCallingCell.js

```diff
@@ -88,6 +88,15 @@ z.components.ConversationListCallingCell = class ConversationListCallingCell {
       return !hasOtherOngoingCalls && isInMinimizedState && (hasPreJoinVideo || isOngoingVideoCall);
     });
 
+    this.showNoCameraPreview = ko.computed(() => {
+      const permissionRepository = this.mediaRepository.streamHandler.permissionRepository;
+      return (
+        !this.showVideoPreview() &&
+        !this.isConnected() &&
+        permissionRepository.permissionState.camera() !== z.notification.PermissionState.GRANTED
+      );
```

Comment:
Hard to read. Mabe we should pull the last condition into const for this to be much easier to read.